hack/make-release-artifacts.sh

```diff
@@ -117,8 +117,6 @@ main() {
     mk_kubernetes_manifests > "${k8s_manifests_file}"
     log "Written ${k8s_manifests_file}"
 
-    cp ${k8s_manifests_file} ../kustomize/base
-
```

Comment:
We don't generate a single, all-encompassing `/kustomize/base/kubernetes-manifests.yaml` file.
In https://github.com/GoogleCloudPlatform/microservices-demo/pull/1134, I added automation to generate the ~11 YAML files inside `/kustomize/base/` when a release is created.

Note: we still generate `/release/kubernetes-manifests.yaml` for anyone that still needs a single file.